### PR TITLE
Release: 1.1.0 (Mainnet)

### PR DESCRIPTION
**Pull request by Shadowfiend**
Rejigger of #731 with proper branch name and dropping the smallest lot size.

### CODE REVIEW

**Comment by Shadowfiend** [comment]
> revert Deposit too small -- Reason given: Deposit too small.

Haha okay, well that's what happens when your tests assume something about initial lot sizes. I'll fix this up as we do the deploy.

**Review by pdyraga — APPROVED** [review]

**Comment by pdyraga** [comment]
Can I have your approval @Shadowfiend on the changes I made?

File: solidity/contracts/test/system/TBTCSystemStub.sol [hunk 1]
```diff
@@ -12,6 +12,7 @@ contract TBTCSystemStub is TBTCSystem {
         TBTCSystem(_priceFeed, _relay)
     public {
         // solium-disable-previous-line no-empty-blocks
+        lotSizesSatoshis = [10**5, 10**6, 10**7, 2 * 10**7, 5 * 10**7, 10**8]; // [0.01, 0.1, 0.2, 0.5, 1.0] BTC
```
Comment (Shadowfiend) [inline solidity/contracts/test/system/TBTCSystemStub.sol:15]: Incidentally, we may not need this.

File: solidity/test/helpers/fullDeployer.js [hunk 2]
```diff
@@ -153,6 +153,10 @@ async function deployAndLinkAll(additions = [], substitutions = {}) {
     1,
   )
 
+  await tbtcSystem.beginLotSizesUpdate([100000, 1e8])
```
Comment (Shadowfiend) [inline solidity/test/helpers/fullDeployer.js:156]: Might be worth dropping a comment on why we're doing this 🤔

**Comment by Shadowfiend** [comment]
Since the two bits are test-related, let's take care of them in a separate PR.

**Pull request merged**